pkg/utils/etcdutil/etcdutil_test.go

```diff
@@ -199,3 +199,36 @@ func TestEtcdKVPutWithTTL(t *testing.T) {
 	re.NoError(err)
 	re.Equal(int64(0), resp.Count)
 }
+
+func TestInitClusterID(t *testing.T) {
```

Comment:
Please ignore change in this file. It will merge with https://github.com/tikv/pd/pull/5985 once it's committed.